schema.sql

```diff
@@ -44,4 +44,14 @@ INSERT INTO
     neutered,
     weight_kg
   )
-VALUES ('Squirtle', 'April 2, 1993', 3, FALSE, -12.13);
\ No newline at end of file
+VALUES ('Squirtle', 'April 2, 1993', 3, FALSE, -12.13);
+
+INSERT INTO
+  animals (
+    NAME,
+    date_of_birth,
+    escape_attempts,
+    neutered,
+    weight_kg
+  )
+VALUES ('Angemon', 'June 12, 2005', 1, TRUE, 45);
```

Comment:
- [ ] Good job so far 👍 Kindly provide the correct value for the weight of Angemon which is `-45` as requested by the previous reviewer 😄

Reply:
thanks , i did fix but its last chance